connection/cluster/ClusterClient.java

```diff
@@ -330,23 +321,17 @@ private TypeDBClientException clusterNotAvailableException() {
     static class FailsafeTaskParameter {
 
         private final ClusterServerClient client;
-        private final ClusterServerStub stub;
         private final ClusterDatabase.Replica replica;
 
-        public FailsafeTaskParameter(ClusterServerClient client, ClusterServerStub stub, ClusterDatabase.Replica replica) {
+        public FailsafeTaskParameter(ClusterServerClient client, ClusterDatabase.Replica replica) {
```

Comment:
We should rename this to `FailsafeTaskParams`, it encapsulates multiple parameters

Reply:
Right. Fixed in https://github.com/vaticle/typedb-client-java/pull/347/commits/ce28757ef2868534e6fda703990f118cf0a3d531.